api/src/pcapi/routes/institutional/playlist.py

```diff
@@ -22,7 +22,7 @@ def get_offers_by_tag(tag_name: str) -> serializers.OffersResponse:
         .join(offers_models.Offer.stocks)
         .filter(
             criteria_models.Criterion.name == tag_name,
-            offers_models.Offer.is_released_and_bookable,
+            offers_models.Offer._released,
```

Comment:
Je pense qu'il faudrait plutôt : 
```suggestion
            offers_models.Offer.isReleased,
```
qui s'assure également que l'offreur a été approuvé : 
```
    @property
    def isReleased(self) -> bool:
        offerer = self.venue.managingOfferer
        return self._released and offerer.isActive and offerer.isValidated
```
Sinon, tu risques d'afficher sur le site instit' des offres qui sont certes "approuvées" mais de partenaires culturels qui n'ont pas encore été validés par l'équipe homologation, ou encore de partenaires culturels qui ont été suspendus par l'équipe conformité (offerer.isActive = False).